src/styles/partials/components/_form.scss

```diff
@@ -1,12 +1,18 @@
 .dg_form-field {
-	input,
-	label,
-	label > span {
-		display: block;
-	}
+  input,
+  label,
+  label > span {
+    display: block;
+  }
 
-	label,
-	.dg_label {
-		margin-bottom: $margin-smallest;
-	}
+  label,
+  .dg_label {
+    margin-bottom: $margin-smallest;
+  }
+}
+
+::placeholder {
+  font-family: $default-body-font;
+  font-size: $small-heading-font-size;
```

Comment:
@tmccoy529 i just removed this when @sdurphy confirmed the font-size should be inherited.